pages/reg_page.php

```diff
@@ -1,84 +1,113 @@
-    <link href="https://fonts.googleapis.com/css2?family=Abel&display=swap" rel="stylesheet">
+    <?php require "../sections/head.php" ?>
     <link rel="stylesheet" href="/css/style_reg_page.css">
     <link rel="stylesheet" href="/css/header-footer.css">
-    <link rel="icon" href="/img/logo.png" type="image/png">
+    <link rel="stylesheet" href="/css/same_styles_log-reg.css">
     <title>Registration</title>
-
 </head>
 
 <body>
 
     <div>
-        <header>
-            <div>
-                <a class="logotype" href="/"><img src="/img/logo.png" class="img-logo"></a>
+        <?php require "../sections/header.php" ?>
+        <?php
+            if (isset($_POST['username'])) {
+                $username = stripslashes($_REQUEST['username']);
+                $username = $mysql->real_escape_string($username);
+                $email = stripslashes($_REQUEST['email']);
+                $email = $mysql->real_escape_string($email);
+                $confirm_email = stripslashes($_REQUEST['confirm_email']);
+                $confirm_email = $mysql->real_escape_string($confirm_email);
+                $password = stripslashes($_REQUEST['password']);
+                $password = $mysql->real_escape_string($password);
+                $confirm_password = stripslashes($_REQUEST['confirm_password']);
+                $confirm_password = $mysql->real_escape_string($confirm_password);
+                $name = stripslashes($_REQUEST['name']);
+                $name = $mysql->real_escape_string($name);
+                $surname = stripslashes($_REQUEST['surname']);
+                $surname = $mysql->real_escape_string($surname);
+                $birthday = stripslashes($_REQUEST['birthday']);
+                $birthday = $mysql->real_escape_string($birthday);
+                $country = stripslashes($_REQUEST['country']);
+                $country = $mysql->real_escape_string($country);
+                $native_city = stripslashes($_REQUEST['native_city']);
+                $native_city = $mysql->real_escape_string($native_city);
 
-                <a class="logo-text" href="/">HL Blog</a>
-            </div>
-            <nav class="menu">
-                <a class="header-links" href="#">News</a>
-                <a class="header-links" href="#">Events</a>
-                <a class="header-links" href="#">Shop</a>
-                <a class="header-links" href="#">Contacts</a>
-            </nav>
-            <div class="header-buttons">
-                <a class="log-reg" href="/pages/log_page.php">Login</a>
-                <a class="log-reg" href="/pages/reg_page.php">Registration</a>
-            </div>
-        </header>
+                $isFormValid = TRUE;
+                if(strlen($username) < 3) {
+                    echo '<div class="container">
+                            <h2 class="h2">Username is too short.</h2>
+                        </div>';
+                    $isFormValid = FALSE;
+                    exit();
```

Comment:
Why do you use `exit()` here? You either use `$isFormValid` variable or `exit();`, using both is useless